pkg/networkservice/core/chain/client.go

```diff
@@ -18,15 +18,15 @@ package chain
 
 import (
 	"github.com/networkservicemesh/api/pkg/api/networkservice"
-	"github.com/sirupsen/logrus"
 
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/trace"
+	"github.com/networkservicemesh/sdk/pkg/tools/logger"
 )
 
 // NewNetworkServiceClient - chains together a list of networkservice.NetworkServiceClient with tracing
 func NewNetworkServiceClient(clients ...networkservice.NetworkServiceClient) networkservice.NetworkServiceClient {
-	if logrus.GetLevel() == logrus.TraceLevel {
+	if logger.IsTracingEnabled() {
```

Comment:
I think we need to create wrapped every time and just pass null logger, so it will be more easy to achieve turn on/off logging per individual request.